includes/class-plugin.php

```diff
@@ -747,7 +916,7 @@ public function maybe_print_comment() {
         );
 
         if ( ! WP_DEBUG ) {
-            printf( "\n<!-- %s -->\n", $message );
+            printf( "\n<!-- %s -->\n", esc_html( $message ) );
```

Comment:
It's a comment, not HTML 🙂

Reply:
Any suggestion for another output sanitization function? In case we decide to translate the comment `esc_html` would be needed.

Reply:
We won't translate the output, since it's used by services to identify the plugin.